src/plugins/search/index.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-unused-vars */
```

Comment:
Let's remove all unused vars if any (note I'm reading this in the GitHub UI, not sure if it was only a spurious comment)

Reply:
ESLint's `no-unused-vars` is configured in `eslint.config.js`. The comment is auto-removed when running `lint:fix` because it is unnecessary. The rule is still enforced.